src/main/java/com/rocketden/main/exception/UserError.java

```diff
@@ -0,0 +1,24 @@
+package com.rocketden.main.exception;
+
+import com.rocketden.main.exception.api.ApiError;
+import com.rocketden.main.exception.api.ApiErrorResponse;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public enum UserError implements ApiError {
+
+    INVALID_USER(HttpStatus.BAD_REQUEST, "Please provide a user with a valid nickname (1-16 characters without spaces).");
+
+    private final HttpStatus status;
+    private final ApiErrorResponse response;
+
+    UserError(HttpStatus status, String message) {
```

Comment:
Can you add an access level (`private`, `protected`, or `public`) here? (The same applies for `RoomError`.)

Reply:
Tried adding an access level but public/protected is not allowed and private gives this message

<img width="525" alt="Enum private is redundant" src="https://user-images.githubusercontent.com/24768574/93647291-ee92fe00-f9bc-11ea-855c-b3f55d56d063.png">

Reply:
Ah ok sorry about that, still learning about `enum`s -- thanks for the clarification there.

Reply:
Yeah same haha, when I first was implementing it I tried to make it public which obviously didn't work out